improve debug information in actions errors response (close #4031) (#4432)
hasura/graphql-engine: docs/graphql/manual/actions/debugging.rst

.. meta::
   :description: Deriving Hasura actions
   :keywords: hasura, docs, actions, debug, debugging

.. _debugging_actions:

Debugging actions
=================

.. contents:: Table of contents
  :backlinks: none
  :depth: 1
  :local:

While you're developing actions for your application, you may want to debug the webhook

> configured for the action. Start the server in :ref:`debugging mode <errors-debugging>`.

Reviewer: ```suggestion
configured for the action. To do so, start the server in :ref:`debugging mode <errors-debugging>`.
```

> In case of any errors the GraphQL response contains debugging information of webhook

Reviewer: ```suggestion
In the case of errors, the GraphQL response contains debugging information of webhook
```

> calls in ``extensions.internal`` field.

Reviewer: ```suggestion
calls in the ``extensions.internal`` field.
```

> For example:

Reviewer: ```suggestion
For example, let's consider the following mutation:
```

.. code-block:: graphql

      mutation {
        create_user(email: "foo@bar.com", name: "Foo"){
          id
          user {
            name
            email
            is_admin
          }
        }
      }

> In case of errors, the reponse will look like the following in debugging mode.

Reviewer: ```suggestion
In case of errors, the response will look like the following if the debugging mode is enabled.
```
